### PR TITLE
Grant contributions

### PR DESCRIPTION
**Pull request by rubelux**
Here is the code for making single payment or donations

### CODE REVIEW

File: pages/settings/[section].js [hunk 1]
```diff
@@ -90,14 +83,29 @@ class Profile extends Component {
...
-        const { pathname, enqueueSnackbar, isMobile } = this.props
+        const { pathname, enqueueSnackbar, isMobile, router } = this.props
+        const { animating } = this.state
 
         return (
             <StartPageM
                 fullScreen={false}
                 {...this.props}
                 render={({ user, pathname, isMobile, ...rest }) => {
+                    console.log('user   ', user)
```
Comment (nobletom123) [inline pages/settings/[section].js:108]: Console log needed?

File: components/Grant/Support/BankdetailsForm.js [hunk 2]
```diff
@@ -169,6 +169,11 @@ class BankdetailsForm extends Component {
             cardlast4Digits,
         } = this.state
 
+        console.log(
```
Comment (nobletom123) [inline components/Grant/Support/BankdetailsForm.js:172]: console log needed?

File: components/Grant/Support/TiersBox.js [hunk 3]
```diff
@@ -58,6 +58,9 @@ const TiersBox = ({
         deselectAll && null
     )
 
+    console.log('showShowMore    ', showShowMore)
+    console.log('showMoreActived', showMoreActived)
```
Comment (nobletom123) [inline components/Grant/Support/TiersBox.js:62]: Console logs needed?

File: components/Lib/CheckUser.js [hunk 4]
```diff
@@ -30,6 +30,7 @@ const CheckUser = ({ auth0User, render }) => {
         skip: !auth0User,
         fetchPolicy: 'network-only',
         onError: () => {
+            console.log('error.graphQLErrors  ', error)
```
Comment (nobletom123) [inline components/Lib/CheckUser.js:33]: console log need?

File: components/Settings/SupportingList.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    },
+}))
+
+const SupportingList = ({
+    containerWidth,
+    currentUser: {
+        grantSubscription: { items: subscriptions },
+    },
+}) => {
+    console.log('subscriptions', subscriptions)
```
Comment (nobletom123) [inline components/Settings/SupportingList.js:31]: console log needed?

File: components/Settings/SupportingList.js [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+                            ({
+                                id,
+                                tierPaymentCycle,
+                                grant: { title, titleURL },
+                                tier: { annualPrice, monthPrice },
+                            }) => {
+                                console.log(
+                                    'tierPaymentCycle   ',
+                                    tierPaymentCycle
+                                )
```
Comment (nobletom123) [inline components/Settings/SupportingList.js:56]: console log needed?

File: components/Lib/CheckUser.js (hunk 4 above)
Comment (rubelux) [reply components/Lib/CheckUser.js:33]: onError console are the only case where they are valid

Comment (rubelux) [reply components/Lib/CheckUser.js:33]: im going to add a Eslint comnent

**Comment by rubelux** [comment]
@nobletom123 logs removed

**Review by nobletom123 — APPROVED** [review]

**Pull request merged**